composer.json

```diff
@@ -1,33 +1,44 @@
 {
 	"name": "wordpress/abilities-api",
-	"description": "AI Abilities for WordPress",
+	"description": "AI Abilities for WordPress.",
 	"license": "GPL-2.0-or-later",
 	"type": "wordpress-plugin",
-	"version": "0.1.0",
 	"keywords": [
 		"ai",
+		"api",
 		"llm",
 		"abilities",
-		"abilities-api",
 		"wordpress"
 	],
-	"homepage": "https://github.com/WordPress/abilities-api/issues",
+	"authors": [
+		{
+			"name": "WordPress AI Team",
+			"homepage": "https://make.wordpress.org/ai/"
+		}
+	],
+	"homepage": "https://github.com/WordPress/abilities-api",
 	"support": {
-		"issues": "https://github.com/WordPress/abilities-api/issues"
+		"issues": "https://github.com/WordPress/abilities-api/issues",
+		"source": "https://github.com/WordPress/abilities-api"
+	},
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"autoload": {
+		"files": [
+			"includes/bootstrap.php"
+		]
 	},
 	"config": {
-		"sort-packages": true,
-		"platform": {
-			"php": "7.4"
-		},
 		"allow-plugins": {
 			"dealerdirect/phpcodesniffer-composer-installer": true,
 			"phpstan/extension-installer": true,
 			"composer/installers": true
-		}
+		},
```

Comment:
I moved a few things around, so the order is similar to https://github.com/WordPress/php-ai-client/blob/trunk/composer.json. That made auditing easier for me.